src/uct/api/uct_def.h

```diff
@@ -210,6 +210,176 @@ typedef struct uct_cm_remote_data {
 } uct_cm_remote_data_t;
 
 
+/**
+ * @ingroup UCT_CLIENT_SERVER
+ * @brief Server endpoint's connect callback arguments field mask.
```

Comment:
Field mask flags for server-side connection established callback
also pls move this enum+struct to later place in uct_def.h so all the structs will appear in the order that events happen in the protocol

Reply:
they currently appear in the order of the protocol - bottom to up (server connect_cb happens last and appears first). you mean to have it the other way around?

Reply:
yes